MapboxNavigation/Error.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+extension NSError {
+    /**
+     Creates a custom `Error` object.
+     */
+    public static func withCustom(message: String, errorCode: SpokenVoiceError = .defaultError) -> NSError {
```

Comment:
Make this method an initializer instead: `NSError(localizedFailureReason:code:)`